### PR TITLE
Updated the submit.py file to now work with Molpro on Zeus

### PR DESCRIPTION
**Pull request by calvinp0**

### CODE REVIEW

**Review by Lilachn91 — APPROVED** [review]

File: Servers/Zeus/ARC/.arc/submit.py [hunk 1]
```diff
@@ -6,7 +7,7 @@
 
 submit_scripts = {
     'local': {
-        'gaussian': """#!/bin/bash -l
+        'gaussian': """#!/bin/sh
```
Comment (alongd) [inline Servers/Zeus/ARC/.arc/submit.py]: Why `sh` instead of `bash`?

File: Servers/Zeus/ARC/.arc/submit.py [hunk 2]
```diff
@@ -36,7 +39,33 @@
...
+
+molpro -n {cpus} -d $MOLPRO_SCRDIR input.in
+
+cp input.* "$PBS_O_WORKDIR/"
+cp geometry.* "$PBS_O_WORKDIR/"
+       
+rm -rf $MOLPRO_SCRDIR    
+        """
+        },
+}
```
Comment (alongd) [inline Servers/Zeus/ARC/.arc/submit.py]: Cשמ טםו שגג שמ קצפאט ךןמק שא איק קמג?

File: Servers/Zeus/ARC/.arc/submit.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 submit_scripts = {
     'local': {
-        'gaussian': """#!/bin/sh
+        'gaussian': """#!/bin/bash
```
Comment (alongd) [inline Servers/Zeus/ARC/.arc/submit.py]: can you squash this commit into the previous one?

Comment (alongd) [reply Servers/Zeus/ARC/.arc/submit.py]: Also, I think we should do `#!/bin/bash -l`

File: Servers/Zeus/ARC/.arc/submit.py [hunk 4]
```diff
@@ -65,7 +67,10 @@
 cp input.* "$PBS_O_WORKDIR/"
 cp geometry.* "$PBS_O_WORKDIR/"
        
-rm -rf $MOLPRO_SCRDIR    
+rm -rf $MOLPRO_SCRDIR
```
Comment (alongd) [inline Servers/Zeus/ARC/.arc/submit.py:75]: squash?

**Review by alongd — COMMENTED** [review]
Thanks! Added some minor comments

**Comment by alongd** [comment]
@Laxzal , looking good. Can you squash?

**Pull request merged**